CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - `defaults/main.yaml`,`tasks/plugins.yml`: Fix Python 3.X compatability issue when checking the contents of sensu_remote_plugins. (@danragnar)
 
 ### Added
-- `templates/sensu-api-json.j2`, `templates/uchiwa_config.json.j2`: Check for explicitly defining uchiwa_users and sensu_api_user_name as empty to disable authentication, useful when having a reverse proxy handling auth in front of the API and/or the uchiwa dashboard (@danragnar)
+- `templates/sensu-api-json.j2`, `templates/uchiwa_config.json.j2`: Check for explicitly defining sensu_uchiwa_users and sensu_api_user_name as empty to disable authentication, useful when having a reverse proxy handling auth in front of the API and/or the uchiwa dashboard (@danragnar)
```

Comment:
For this bit, instead of editing the older change entry. Could you add something to the `[Unreleased]` section? I'm still going through your work here but, I _think_ we'll need to cut this as a `Breaking Change`. Check out https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md for some details. Let me know if anything doesn't make sense!

Reply:
So yeah, I've gone through this once now, definitely will need to be called out as a breaking change since we will need people to update their variable names. I was previously talking to @cjchand about #16 and wanted to see if there was some way where we could alias the existing variables to the new ones, to get the new names in there without a breaking change, and then cut the breaking one and purge the older, unprefixed names. 

That all said, perhaps doing that amount of magic isn't worth it. Curious what folks think. Overall, this looks solid to me, and the integration tests pass! \o/ Thanks so much for knocking out the hard work here.

Reply:
Was wondering about changing older references, but then thought about it - and I wanted to the match the variable names/references with the new ones so it wouldn't be a case of "where did that variable go" confusion?

Reply:
Oh boy, good point. I was thinking, perhaps it'd be nice to like dump the list of changed variables, and document that on the CHANGELOG/Github Release. 

Something like:


### Breaking Change:
* All variables for this role are now prefixed with `sensu_` in order to support integrations with other roles and avoid namespace collisions. If you have customized any of the variables used in this role, please ensure that you have updated their names. We've included the changed variable names below.
```
se_enterprise -> sensu_se_enterprise
se_user -> sensu_se_user
se_pass -> sensu_se_pass
```


That said, maybe that's a bit over the top.